enclave_manager/avalon_enclave_manager/wpe/wpe_enclave_manager.py

```diff
@@ -42,6 +43,10 @@ class WorkOrderProcessorEnclaveManager(EnclaveManager):
 
     def __init__(self, config):
         super(WorkOrderProcessorEnclaveManager, self).__init__(config)
+        # Calculate sha256 of enclave id to get 32 bytes. Then take a
+        # hexdigest for hex str.
+        enclave_id_utf = self.enclave_id.encode("UTF-8")
+        self._enclave_id_hex = hashlib.sha256(enclave_id_utf).hexdigest()
```

Comment:
Are we using `_enclave_id_hex ` this in this PR ? If not, can we remove it?

Reply:
Not using. Removed.